### ISSUE TITLE
Cannot 'add_whitelist' unless 'enable=' or 'reset_whitelist' is called first

### ISSUE THREAD

**Issue by 0x0badc0de** [flyerhzm/bullet#252]
Moved 'add_whitelist' into separate file and have following issue:
gems/bullet-4.14.7/lib/bullet.rb:77:in `add_whitelist': undefined method `[]' for nil:NilClass (NoMethodError)
If I call 'enable=' or 'reset_whitelist' first - everything is find.
See that whitelist gets reset when 'enable=' is invoked so it becomes harder to have separate whitelist population and per-env bullet configuration. Can it worth to init whitelist in initializer and don't touch in 'enable='?

**Comment by openbl** [comment]
I just ran into the same issue. Setting `Bullet.enable = true` before adding to the whitelist fixed my problem.

An interesting note is I had originally called `Bullet.enable = true` in each environment configuration file mostly for environment-dependent notifications; however, I moved the call to `Bullet.enable = true` (and the whitelisted query) to `config/application.rb` since calling `Bullet.enable = true` after adding to the whitelist will clear out previously added whitelists.
